src/pdn/src/pdn.tcl

```diff
@@ -854,7 +858,7 @@ proc define_pdn_grid { args } {
   sta::parse_key_args "define_pdn_grid" args \
     keys {-name -voltage_domains -pins -starts_with -obstructions -power_switch_cell \
       -power_control -power_control_network} \
-    flags {}
+    flags {};# checker off
```

Comment:
public command

Reply:
For this, I turned off checking because its flags are spread over different sub-functions: `define_pdn_grid_macro`, `define_pdn_grid_existing` and `define_pdn_grid`. I think this case is pretty rare, we can fix in a future PR maybe?